Reference/Mapping/index-vpre8.1.md

```diff
@@ -0,0 +1,7 @@
+---
+versionTo: 8.1.0
+---
+
+# Mapping in Umbraco
```

Comment:
Mapping objects from one type to another in Umbraco

or

Object Mapping in Umbraco

or

Model Mapping in Umbraco